pkg/logutil/log.go

```diff
@@ -30,23 +30,35 @@ import (
 const (
 	defaultLogTimeFormat = "2006/01/02 15:04:05"
 	defaultLogMaxSize    = 300 // MB
-	defaultLogMaxBackups = 3
-	defaultLogMaxAge     = 28 // days
+	defaultLogFormat     = "text"
 	defaultLogLevel      = log.InfoLevel
 
 	logDirMode = 0755
 )
 
 // FileLogConfig serializes file log related config in toml/json.
 type FileLogConfig struct {
+	// Log filename, leave empty to disable file log.
 	Filename string `toml:"filename" json:"filename"`
+	// Is log rotate enabled. TODO.
+	LogRotate bool `toml:"log-rotate" json:"log-rotate"`
+	// Max size for a single file, in MB.
+	MaxSize int `toml:"max-size" json:"max-size"`
+	// Max log keep days, default is never deleting.
+	MaxDays int `toml:"max-days" json:"max-days"`
+	// Maximum number of old log files to retain.
+	MaxBackups int `toml:"max-backups" json:"max-backups"`
 }
 
 // LogConfig serializes log related config in toml/json.
 type LogConfig struct {
 	// Log level.
 	Level string `toml:"level" json:"level"`
-	// Log file.
+	// Log format. one of json, text, or console.
+	Format string `toml:"format" json:"format"`
+	// Disable automatic timestamps in output.
+	DisableTimestamp bool `toml:"disable-timestamp" json:"disable-timestamp"`
```

Comment:
ELK system will add a timestamp and it has more accuracy.
Enduser can also translate this timestamp in `time` field via logstash.